readme.md

```diff
@@ -103,26 +103,29 @@ npx truffle migrate
 yarn test
 ```
 
+### Lint
 
-### Break down into end to end tests
+ESLint will be run automatically before you commit. To run it manually:
 
-Explain what these tests test and why
 
 ```
-Give an example
+yarn lint:fix
 ```
 
-### And coding style tests
+## Deployment
 
-Explain what these tests test and why
+The code is deployed automatically on each push to GitHub. A GitHub action will create a test website and post the link as a comment in the PR.
 
-```
-Give an example
-```
+When pushing to the `master` branch, the code will be automatically deployed to production.
```

Comment:
This is not 100% true, the build will be uploaded but not published to the production, publishing is done manually by the devops. We did it this way to avoid unexpected pushes as we kind of have a security critical product

Reply:
Yeah, I wasn't sure, thanks for this correction!